budgetr-backend/src/main/java/pl/mantiscrab/budgetr/domain/bankaccount/BankAccount.java

```diff
@@ -0,0 +1,28 @@
+package pl.mantiscrab.budgetr.domain.bankaccount;
+
+import lombok.*;
+import pl.mantiscrab.budgetr.domain.user.User;
+
+import javax.persistence.*;
+import javax.validation.constraints.Digits;
+import java.math.BigDecimal;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter(AccessLevel.PACKAGE)
+@Entity
+@Table(name = "BANK_ACCOUNT")
+public class BankAccount {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Setter(AccessLevel.PACKAGE)
+    private Long id;
+    private String name;
+
+    @ManyToOne
```

Comment:
Many to one why?

Reply:
Budgetr application goal is to help users with budgeting. One user can have multiple bank accounts related (we know it from real life, right?) so I also implemented it in code